marlowe-dashboard-client/src/Play/View.purs

```diff
@@ -58,9 +58,9 @@ renderHeader walletNickname menuOpen =
             , span
                 [ classNames $ [ "hidden", "md:flex", "md:items-baseline" ] <> Css.button <> [ "bg-white" ] ]
                 [ span
-                    [ classNames $ [ "-m-1", "mr-2", "rounded-full", "text-white", "w-5", "h-5", "flex", "justify-center", "items-center", "uppercase" ] <> Css.bgBlueGradient ]
+                    [ classNames $ [ "-m-1", "mr-2", "rounded-full", "text-white", "w-5", "h-5", "flex", "justify-center", "items-center", "uppercase", "font-semibold" ] <> Css.bgBlueGradient ]
                     [ text $ take 1 walletNickname ]
-                , text walletNickname
+                , span [ classNames [ "truncate", "w-16" ] ] [ text walletNickname ]
```

Comment:
I think this needs to be `"min-w-16"` (which we'll have to add to the tailwind config). Otherwise the button is too wide for smaller nicknames.

Reply:
good catch